### PR TITLE
Improve assist dialog

### PR DESCRIPTION
**Pull request by piitaya**
## Proposed change

Text is displayed after clicking the microphone icon

image

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by bramkragten** [comment]
Should the color of the message bubble be in the warning or error color?

File: src/translations/en.json [hunk 1]
```diff
@@ -838,7 +838,11 @@
         "input_label": "Enter a request",
         "send_text": "Send text",
         "start_listening": "Start listening",
-        "manage_assistants": "Manage assistants"
+        "manage_assistants": "Manage assistants",
+        "not_supported_microphone": "Microphone is not supported. You need to access from a secure URL to use it. ",
```
Comment (bramkragten) [inline src/translations/en.json:842]:
```suggestion
        "not_supported_microphone": "Microphone is not supported. You need to access Home Assistant from a secure URL to use it. ",
```

File: src/dialogs/voice-command-dialog/ha-voice-command-dialog.ts [hunk 2]
```diff
@@ -382,14 +397,53 @@ export class HaVoiceCommandDialog extends LitElement {
...
       this._startListening();
     } else {
       this._stopListening();
     }
   }
 
+  private async _showNotSupportedMessage() {
+    this._addMessage({
+      who: "hass",
+      text: html`
```
Comment (bramkragten) [inline src/dialogs/voice-command-dialog/ha-voice-command-dialog.ts:418]:
```suggestion
      who: "hass",
      error: true,
      text: html`
```

**Comment by matthiasdebaat** [comment]
> Should the color of the message bubble be in the warning or error color?

Think blue is good enough. Makes it more of a conversation with a bot.

**Comment by bramkragten** [comment]
> > Should the color of the message bubble be in the warning or error color?
> 
> Think blue is good enough. Makes it more of a conversation with a bot.

But it is not? It is the response of pressing a button that does not do what you would expect it to do?

**Review by bramkragten — DISMISSED** [review]

File: src/dialogs/voice-command-dialog/ha-voice-command-dialog.ts [hunk 3]
```diff
@@ -401,7 +401,7 @@ export class HaVoiceCommandDialog extends LitElement {
     ev.stopPropagation();
     ev.preventDefault();
     const supportsMicrophone = AudioRecorder.isSupported;
-    if (!supportsMicrophone) {
+    if (supportsMicrophone) {
```
[the remark this answers is not in the document]
Comment (piitaya) [reply src/dialogs/voice-command-dialog/ha-voice-command-dialog.ts:404]: 🤦‍♂️

**Review by bramkragten — APPROVED** [review]

**Pull request merged**